api/java/src/main/java/com/qbot/answeringservice/controller/FileUploadController.java

```diff
@@ -0,0 +1,74 @@
+package com.qbot.answeringservice.controller;
+
+import java.io.IOException;
+import java.util.stream.Collectors;
+
+import com.qbot.answeringservice.exception.StorageFileNotFoundException;
+import com.qbot.answeringservice.service.StorageService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.mvc.method.annotation.MvcUriComponentsBuilder;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+@Controller
+@RequestMapping("/files")
+public class FileUploadController {
+
+    private final Logger logger = LoggerFactory.getLogger(FileUploadController.class);
+    private final StorageService storageService;
+
+    @Autowired
+    public FileUploadController(StorageService storageService) {
+        this.storageService = storageService;
+    }
+
+    @GetMapping("/")
```

Comment:
You probably don't need the path parameter in this annotation or the one on line 58